cmd/ops/deprecation.go

```diff
@@ -29,7 +31,14 @@ var (
 func init() {
 	Program.AddCommand(deprecationsCmd)
 	deprecationsCmd.Flags().StringVar(&schemaFile, schemaFileFlag, "", "Server's schema file (required)")
+	deprecationsCmd.Flags().BoolVar(&xcodeFormat, xcodeFormatFlag, false, "Override output format to xcode [EXPERIMENTAL]")
```

Comment:
can't this be another output format? like `--output json`, `--output xcode`

Reply:
I think it works just fine, so I will make it not experimental and just another format as you suggest